src/Dynamic.cpp

```diff
@@ -217,7 +217,7 @@ class PointerData : public hx::Object
    String toString()
    {
       char buf[100];
-      sprintf(buf,"Pointer(%p)", mValue);
+      snprintf(buf,sizeof(buf),"Pointer(%p)", mValue);

```

Comment:
Was this correct? `sizeof()` returns the size of a single `char`, it doesn't return `100`.

Reply:
Actually, it looks like in this case it does work properly since `char buf[]` is stack allocated. Is it also working correctly in `mysql/my_api.cpp`?

Reply:
Would seem like this is also the case there: https://github.com/HaxeFoundation/hxcpp/blob/08f88ff314eeaa165dd76272f92c8030cc879ec2/src/hx/libs/mysql/my_proto.h#L114

Should all be good then.